tests/address_bar_and_search/test_adaptive_history_autofill.py

```diff
@@ -43,29 +43,29 @@ def test_add_adaptive_history_autofill(driver: Firefox):
 
     # Type the first 3 characters of the visited URL in the address bar and select the suggested URL
     nav.type_in_awesome_bar("nat")
-    nav.get_element("firefox-suggest").click()
+    with driver.context(driver.CONTEXT_CHROME):
+        nav.get_element("firefox-suggest").click()
     nav.expect_in_content(
         EC.url_contains("https://www.nationalgeographic.com/science/")
     )
 
-    tabs.set_content_context()
-
     # Open a new tab, type the first 3 characters of the visited URL
     tabs.new_tab_by_button()
     tabs.wait_for_num_tabs(2)
     driver.switch_to.window(driver.window_handles[-1])
     nav.type_in_awesome_bar("nat")
 
-    autofill_adaptive_element = nav.get_element(
-        "search-result-autofill-adaptive-element"
-    )
+    # tabs.set_chrome_context()
```

Comment:
Seems like these commented lines may need to go?

Reply:
oops, uncommented